examples/spark/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+import java.util.Properties
+
+plugins {
+    kotlin("jvm") version "1.3.70"
+    application
+}
+
+allprojects {
+    buildscript {
+        repositories {
+            mavenLocal()
+            jcenter()
+            mavenCentral()
+        }
+    }
+
+    repositories {
+        mavenLocal()
+        jcenter()
+        mavenCentral()
+    }
+}
+
+application {
+    mainClassName = "com.expediagroup.graphql.examples.spark.Application"
+}
+
+dependencies {
+    implementation(kotlin("stdlib", "1.3.70"))
+    compile("com.expediagroup", "graphql-kotlin")
```

Comment:
This is not the library you need to include, the `examples` build script already includes all the depenencies for the sub-projects.

https://github.com/ExpediaGroup/graphql-kotlin/blob/2806cd80125dbba451150e42bbf700589649f923/examples/build.gradle.kts#L45

You need to add the sub-project here: https://github.com/ExpediaGroup/graphql-kotlin/blob/2806cd80125dbba451150e42bbf700589649f923/examples/settings.gradle.kts#L25

Reply:
i think we'll have to change common dependencies to only include schema generator, for now you could exclude the spring server dependency and include schema gen explicitly

Reply:
Thanks, I wasn't catching the subprojects declaration.